GirafRest.Test/Controllers/UserControllerTest2021.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System;
+using System.Collections.Generic;
+using GirafRest.Controllers;
+using System.Text;
+using Xunit;
+using Moq;
+using Microsoft.AspNetCore.Http;
+using Microsoft.EntityFrameworkCore;
+using GirafRest.Extensions;
+using GirafRest.Models;
+using GirafRest.Models.DTOs;
+using GirafRest.Models.Responses;
+using GirafRest.Services;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using System.Linq;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+using Microsoft.EntityFrameworkCore.InMemory;
+using GirafRest.Data;
+using GirafRest.Test.Mocks;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace GirafRest.Test.Controllers
+{
+    [TestClass]
+    public class UserControllerTest2021
+    {
+        readonly GirafRoles citizen = GirafRoles.Citizen;
+
+        [TestMethod]
+        public void GetSortedCitizens()
```

Comment:
Denne test tester om en liste bliver sortret korrekt, og derfor indirekte tester implementation af ```IComparable<DisplayNameDTO>```. Men jeg vil ikke mene at alle [cases](https://docs.microsoft.com/en-us/dotnet/api/system.icomparable.compareto?view=net-5.0#notes-to-implementers) blvier testet her. Jeg vil mene at den implementerede funktion ```CompareTo```` skal unit testes (Ikke i konteksten af lister).

Reply:
er det ikke det samme, for at istedet at teste om den returnere 0 eller 1 efter en sortering, så skal den kunne finde ud af at sortere elementer, hvilket er det den gør... hmm